src/pages/zh-cn/core-concepts/framework-components.md

```diff
@@ -94,7 +98,7 @@ import InteractiveCounter from '../components/InteractiveCounter.jsx';
 
 你可以在同一个 Astro 组件中导入并渲染来自多个框架的组件。
 
-```astro
+```astro title="src/pages/component-children.astro" {5}
 ---
 // src/pages/MyAstroPage.astro
```

Comment:
Looks like the code titles got a bit mixed up here. This one should be “mixing frameworks” rather than “component-children”.

````suggestion
```astro
---
// src/pages/mixing-frameworks.astro
````